### PR TITLE
Added development.md

### LINKED ISSUE #853: Split the README.md into README.md and DEVELOPMENT.md
`README.md` ought to focus on the end-user (people who are using the extension) and `DEVELOPMENT.md` ought to focus on the developers who are developing the extension.  Developers are getting confused and would like a "single source of truth" for running a proper build.

### PR DESCRIPTION
**Pull request by nlewis05**
closes #853 

Added development.md

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+https://github.com/apache/daffodil-vscode/wiki/Apache-Daffodil%E2%84%A2-Extension-for-Visual-Studio-Code:-v1.3.1 
+
+## Contributing
+Due to this being an Apache project, if you would like to contribute, you will need to fork the daffodil-vscode main branch to your own repo of choice and create pull requests to the main branch with your code changes.
+ ![image](https://github.com/ctc-oss/daffodil-vscode/assets/131286323/160ff687-4b37-4730-b867-128d063950b7)
+ 
+Once forked, you can clone that forked repository to your own local environment. This can be done by using `git clone`, followed by copy and pasting the https or ssh urls shown under the “Code” dropdown.
+ ![image](https://github.com/ctc-oss/daffodil-vscode/assets/131286323/e32b5ebf-a45d-4362-aa44-5aa817d4bd0b)
+
+ ![image](https://github.com/ctc-oss/daffodil-vscode/assets/131286323/3e123e64-70bb-4f5d-87c9-08f28428630c)
```
Comment (lrbarber) [inline DEVELOPMENT.md:15]: Recommend changing this `git clone` image to one with a generic (i.e., fake) username

**Review by shanedell — APPROVED** [review]
+1. Just rebase off of the Apache repo's `main` branch and squash all commits except one. Then create a message you like and force push up to your branch.

**Comment by nlewis05** [comment]
Ah I see what you mean, I was curious how that worked with the PR and if it would move them over here, but I guess that's not how that works, I'll see about moving them over here

**Comment by nlewis05** [comment]
Seems they may have to be added manually by someone with write access to that wiki file @shanedell

**Comment by stevedlawrence** [comment]
Another option is to replace images with text where possible. For example, the image that shows the fork button could be replaced with the a link to https://github.com/apache/daffodil-vscode/fork. Making things a user can copy paste or click is probably a bit easier than telling people to find a button on another page. Avoiding images where possible is also friendlier to people with visual impairments.

**Pull request merged**